releases/deployment.json

```diff
@@ -60,5 +60,19 @@
       ["0xf5e4C09Cd8A01F066f976A0c17f88e549817ea17", 6944]
     ],
     "DonationsStaking": "0xe58501733711B966f9c1d81B29557674c11bD840"
+  },
+  "fuse": {
```

Comment:
missing contracts
make sure you have everything from contracts,stakingmodel,upgradeables